### PR TITLE
Change CI to Github actions and add Python 2.6 tests

### PR DESCRIPTION
**Pull request by querti**

### CODE REVIEW

**Comment by querti** [comment]
It seems that the tests don't run until being merged. The results can be seen in https://github.com/querti/iiblib/actions/runs/405656674

File: .github/workflows/tox-tests.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      - name: Update existing dependencies
+        run: sudo apt-get update -y
+      - name: Install system dependencies
+        run: sudo apt-get install -y libkrb5-dev
+      - name: Add Python 2.6 repository
+        run: sudo add-apt-repository ppa:deadsnakes/ppa
+      - name: Install Python 2.6
+        run: sudo apt-get install -y python2.6 python2.6-dev
+      - name: Install pip
+        run: curl https://bootstrap.pypa.io/2.6/get-pip.py -o get-pip.py; sudo python2.6 get-pip.py
```
Comment (lipoja) [inline .github/workflows/tox-tests.yml:19]: just small question. Would not be better to use &&:
`curl https://bootstrap.pypa.io/2.6/get-pip.py -o get-pip.py && sudo python2.6 get-pip.py`

File: .github/workflows/tox-tests.yml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+          python-version: 3.7
+      - name: Install Tox
+        run: pip install tox
+      - name: Run Tox
+        run: tox -e py37
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1.0.15
+        with:
+          fail_ci_if_error: true
+          verbose: true
```
Comment (lipoja) [inline .github/workflows/tox-tests.yml:124]: Do we care about the new lines at the end of file?

File: .github/workflows/tox-tests.yml (hunk 1 above)
Comment (querti) [reply .github/workflows/tox-tests.yml]: Changed.

File: .github/workflows/tox-tests.yml (hunk 2 above)
Comment (querti) [reply .github/workflows/tox-tests.yml]: Added.

**Review by lipoja — APPROVED** [review]

**Comment by querti** [comment]
@release-engineering/distribution pls review

**Comment by querti** [comment]
Abandoning, as Travis functionality has been restored and Python 2.6 support will be tested by pub.

**Pull request closed without merging**